trajopt_optimizers/trajopt_sqp/src/osqp_eigen_solver.cpp

```diff
@@ -38,7 +38,7 @@ OSQPEigenSolver::OSQPEigenSolver()
   solver_.settings()->setWarmStart(true);
   solver_.settings()->setPolish(true);
   solver_.settings()->setAdaptiveRho(false);
-  solver_.settings()->setMaxIteraction(8192);
+  solver_.settings()->setMaxIteration(8192);
```

Comment:
Was this a typo in their api or was this a valid setting that was doing something else?

Reply:
I think so on their end so it was deprecated.